source/config/configSpec.py

```diff
@@ -335,6 +335,7 @@
 [addonStore]
 	showWarning = boolean(default=true)
 	automaticUpdates = option("notify", "disabled", default="notify")
+	baseURL = string(default="")
```

Comment:
I think this could be a bit more descriptive, perhaps:

```suggestion
	baseServerURL = string(default="")
```